now.json

```diff
@@ -20,6 +26,10 @@
     {
       "source": "/v1/bite.js",
       "destination": "bite.v1.js"
+    },
+    {
+      "source": "/api/robots.txt",
+      "destination": "robots.txt"
```

Comment:
Was also thinking of re-implementing the catch-all route as mentioned [here](https://github.com/barnebys/analytics/pull/9#discussion_r357870926), but wasn't sure of which URL it should point towards

Reply:
I will setup an alias for that tomorrow. Just need to come up with decent name/url  :-)

Reply:
Let me know when that's ready!